WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -583,6 +584,7 @@ private void showPhotoPicker() {
     private void hidePhotoPicker() {
         if (isPhotoPickerShowing()) {
             mPhotoPickerFragment.finishActionMode();
+            mPhotoPickerFragment.setPhotoPickerListener(null);
```

Comment:
I wonder if it would be better to unset the listener _before_ calling `finishActionMode()`?